wrappers/python/pyproject.toml.in

```diff
@@ -6,7 +6,8 @@ build-backend = "setuptools.build_meta"
 name = "moordyn"
 version = "${MOORDYN_VERSION}"
 authors = [
-  { name="Jose Luis Cercos-Pita", email="jlc@core-marine.com" },
+  { name="Jose Luis Cercos-Pita", email="jlc@core-marine.com" }, 
+  { name="Matt Hall", email="matthew.hall@nrel.gov"}
```

Comment:
I am totally OK with this, I mean, it is totally OK to me to share authorship.

However:

- This is an open source project, so the license allows everyone to do almost whatever they want. Thus, adding authors have very little practical effect.
- Usually the authors on a packaging are the ones who are maintaining it (no matter who is developing the packed software). The logic is that users can perfectly know who they should ask to in case of problems.

As I said, totally fine with me, but maybe you @RyanDavies19 and @mattEhall prefer to give it a second thought